### PR TITLE
fix: showTotal should work on simple mode

### PR DESCRIPTION
**Pull request by KevinBrother**
simple 支持 showTotal 的配置选项

### CODE REVIEW

**Comment by yoyo837** [comment]
设计好像技术simple不暂时total.

File: src/Pagination.jsx [hunk 1]
```diff
@@ -425,6 +425,19 @@ class Pagination extends React.Component {
...
+    if (showTotal) {
+      totalText = (
+        <li className={`${prefixCls}-total-text`}>
+          {showTotal(total, [
+            total === 0 ? 0 : (current - 1) * pageSize + 1,
+            current * pageSize > total ? total : current * pageSize,
+          ])}
+        </li>
+      );
+    }
```
Comment (afc163) [inline src/Pagination.jsx:440]:
```suggestion
    const totalText = showTotal && (
        <li className={`${prefixCls}-total-text`}>
          {showTotal(total, [
            total === 0 ? 0 : (current - 1) * pageSize + 1,
            current * pageSize > total ? total : current * pageSize,
          ])}
        </li>
      );
```

Comment (KevinBrother) [reply src/Pagination.jsx]: 优雅

File: src/Pagination.jsx [hunk 2]
```diff
@@ -426,18 +426,14 @@ class Pagination extends React.Component {
...
-      );
-    }
+   const totalText = showTotal && (
+      <li className={`${prefixCls}-total-text`}>
+        {showTotal(total, [
+          total === 0 ? 0 : (current - 1) * pageSize + 1,
+          current * pageSize > total ? total : current * pageSize,
+        ])}
+      </li>
+    );
```
Comment (afc163) [inline src/Pagination.jsx:436]:
```suggestion
    const totalText = showTotal && (
      <li className={`${prefixCls}-total-text`}>
        {showTotal(total, [
          total === 0 ? 0 : (current - 1) * pageSize + 1,
          current * pageSize > total ? total : current * pageSize,
        ])}
      </li>
    );
```

Comment (KevinBrother) [reply src/Pagination.jsx:436]: https://github.com/react-component/pagination/pull/433/commits/80450974d8d38db67c6afdfd438b2e8f62feec17

**Pull request merged**